### PR TITLE
Add duration parameter to average parallactic angle calculation method

### PR DESCRIPTION
**Pull request by toddburnside**

### CODE REVIEW

File: modules/core/shared/src/main/scala/lucuma/core/math/skycalc/package.scala [hunk 1]
```diff
@@ -28,8 +29,9 @@ def parallacticAngle(site: Site, tracking: ObjectTracking, vizTime: Instant): An
   skycalc.calculate(c, vizTime, false).parallacticAngle
 }
 
-def averageParallacticAngle(site: Site, tracking: ObjectTracking, vizTime: Instant): Option[Angle] =
-  val defined: TimeRange = (vizTime.toEpochMilli(), vizTime.toEpochMilli() + 1.hour.toMillis)
+def averageParallacticAngle(site: Site, tracking: ObjectTracking, vizTime: Instant, duration: Duration): Option[Angle] =
+  val end = vizTime.toEpochMilli() + duration.toMillis()
```
Comment (swalker2m) [inline modules/core/shared/src/main/scala/lucuma/core/math/skycalc/package.scala:33]: I think this could also be `vizTime.plus(duration)`.

**Review by swalker2m — APPROVED** [review]
Looks good.  :+1:

File: modules/core/shared/src/main/scala/lucuma/core/math/skycalc/package.scala (hunk 1 above)
Comment (toddburnside) [reply modules/core/shared/src/main/scala/lucuma/core/math/skycalc/package.scala:33]: Yeah. That looks cleaner. Then it's short enough to inline.

**Review by cquiroz — APPROVED** [review]
LGTM

**Pull request merged**